File: tools/figcheck/fixtures/makie/bad.jl
# Deliberately invalid Julia script for figcheck's makie backend self-test:
# plots against a name that was never defined, so Julia must raise
# UndefVarError and exit non-zero.
using CairoMakie

fig = Figure(size = (200, 200))
ax = Axis(fig[1, 1])
lines!(ax, this_name_is_never_defined)

File: tools/figcheck/fixtures/matplotlib/bad.py
"""Deliberately invalid matplotlib script for figcheck's matplotlib backend
self-test: plots against a name that was never defined, so Python must
raise NameError and exit non-zero.
"""

import matplotlib

matplotlib.use("Agg")
import matplotlib.pyplot as plt

fig, ax = plt.subplots()
ax.plot(this_name_is_never_defined)
